CHANGELOG.rst

```diff
@@ -1,6 +1,21 @@
 Changelog
 =========
 
+Version v0.13.0
+---------------
+
+New Features
+~~~~~~~~~~~~~
+- Node/edge populations are now supported in config
+- Population type available in NodePopulation/EdgePopulation
+- Population config (if given) overwrites the "components" config for that population
+- Alternate morphology directories (.h5, .asc) are now supported
+- Added different edge/node types to bluepysnap/bbp.py
```

Comment:
Please adjust this line and it is good to go for me.

Reply:
Right, thanks.